### PR TITLE
NEPT-205: Moved permissions to soft config, added page permissions to…

### PR DESCRIPTION
**Pull request by imanoleguskiza**
… editor, replaced role in behat tests

### CODE REVIEW

File: tests/features/contact_form.feature [hunk 1]
```diff
@@ -23,7 +23,7 @@ Feature: Contact Form
 
   @javascript
   Scenario: Administrator user can submit the contact page
```
Comment (voidtek) [inline tests/features/contact_form.feature]: Must be:
Editor user can submit the contact page

File: tests/features/contact_form.feature [hunk 2]
```diff
@@ -22,8 +22,8 @@ Feature: Contact Form
       | Math question field is required. |
 
   @javascript
-  Scenario: Administrator user can submit the contact page
-    Given I am logged in as a user with the editor role
+  Scenario: Administrator user can submit the gcontact page
```
Comment (voidtek) [inline tests/features/contact_form.feature]: "gcontact" -> "contact"

File: tests/features/text_formats.feature [hunk 3]
```diff
@@ -23,7 +23,7 @@ Feature: Text formats configuration
     And the "anonymous user" checkbox should <anonymous user>
 
     Examples:
-      | format        | administrator | contributor   | editor      | authenticated_user  | anonymous user  |
-      | full_html     | be checked    | be checked    | be checked  | not be checked      | not be checked  |
-      | filtered_html | be checked    | be checked    | be checked  | be checked          | be checked      |
-      | basic_html    | be checked    | be checked    | be checked  | be checked          | be checked      |
+      | format        | administrator | contributor       | editor          | authenticated_user  | anonymous user  |
```
Comment (voidtek) [inline tests/features/text_formats.feature]: Change the order of the columns to understand better the permission checks.

**Comment by netlooker** [comment]
I did some small changes and I've created and updated README.md files. The rest looks ok.

**Comment by netlooker** [comment]
👍

**Comment by dxvargas** [comment]
👍

**Pull request merged**